physics/rrtmgp_aerosol_optics.F90

```diff
@@ -65,34 +57,37 @@ subroutine rrtmgp_aerosol_optics_run(doSWrad, doLWrad, nCol, nLev, nTracer, nTra
     ! Outputs
     real(kind_phys), dimension(:,:), intent(out) :: &
          aerodp                   ! Vertical integrated optical depth for various aerosol species 
-    type(ty_optical_props_2str),intent(out) :: &
-         sw_optical_props_aerosol ! RRTMGP DDT: Longwave aerosol optical properties (tau)
-    type(ty_optical_props_1scl),intent(inout) :: &
-         lw_optical_props_aerosol ! RRTMGP DDT: Longwave aerosol optical properties (tau)
+    real(kind_phys), dimension(:,:,:), intent(out) :: &
```

Comment:
Same comment about uninitialized intent(out) variables. It could be fine; just need to doublecheck. I mean, it wasn't causing problems before, so why should it now?